glide.yaml

```diff
@@ -8,6 +8,7 @@ import:
 - package: github.com/gorilla/context
 - package: github.com/gorilla/mux
 - package: github.com/heketi/rest
+  version: aa6a652074133e6b0836bb095e847021fcb94b6a
```

Comment:
This hunk is a remnant of the original patch that got merged as PR #1133.
This hunk is not needed (as discussed).

@raghavendra-talur please remove this hunk!